neqo-transport/src/connection.rs

```diff
@@ -2286,6 +2347,7 @@ mod tests {
             Rc::new(RefCell::new(FixedConnectionIdManager::new(3))),
             loopback(),
             loopback(),
+            QuicVersion::Draft28,
```

Comment:
For maintenance purposes:

```suggestion
            QuicVersion::default(),
```

We're going to want to test only one version most of the time.

Reply:
done